source/_components/xs1.markdown

```diff
@@ -0,0 +1,112 @@
+---
+layout: page
+title: "EZcontrol XS1"
+description: "Instructions on how to integrate an XS1 Gateway within Home Assistant."
+date: 2018-12-04 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xs1.jpg
+ha_category: 
+  - Hub
+  - Climate
+  - Sensor
+  - Switch
+redirect_from:
+  - /components/climate.xs1/
```

Comment:
I guess we only need the redirects if there were already pages. I think we can remove them. Hopefully the index will still populate.

Reply:
Seems like it doesn't? I don't understand what the build error message is trying to tell me :disappointed:

Reply:
My guess was that it failed du to the redirects looking for pages that didn't exist. But I guess I'm wrong.

@klaasnicolaas any idea?

Reply:
`redirect_from` is only for pages that already exist.

Reply:
@klaasnicolaas I removed that already but it still doesn't compile :cry: